.github/CODEOWNERS

```diff
@@ -972,10 +972,10 @@
 # ServiceOwners:                                                   @jamesfan1 @bangbingsyb @spencerofwiti
 
 # PRLabel: %Oracle
-/sdk/oracle/Azure.ResourceManager.*/                               @v-rvilathurs
+/sdk/oracle/Azure.ResourceManager.*/                               @v-jamcheung @v-rvilathurs @eelhomsi1
```

Comment:
`@eelhomsi1` is not a valid GitHub name.

Reply:
Updated

Reply:
@v-jamcheung please join the https://github.com/orgs/Azure/teams/azure-sdk-partners

Reply:
done